sendto_silhouette.py

```diff
@@ -555,6 +555,11 @@ def multipassOvercut(self, paths, multipass, reversetoggle, overcut):
 
 
     def effect(self):
+        reg_origin_X = self.options.regoriginx
+        reg_origin_Y = self.options.regoriginy
+        reg_width = self.options.regwidth if self.options.regwidth else int(self.svg.get("width").rstrip("mm")) - reg_origin_X*2
+        reg_length = self.options.reglength if self.options.reglength else int(self.svg.get("height").rstrip("mm")) - reg_origin_Y*2
+
```

Comment:
It's a perfect plan to detect regmarks from svg WRT #254. 
NB: those regmark IDs are used allover the places and in other templates as well 
see https://github.com/search?q=repo%3Afablabnbg%2Finkscape-silhouette%20id%3D%22regmark-tl%22&type=code 
i.e.
https://github.com/fablabnbg/inkscape-silhouette/blob/af3c9019e948aa56c9a357263eec45a35df18f20/templates/silhouette-cameo-registration-marks-a4.svg#L149-L160

... and of course this early draft is based on several assumptions
- document settings in mm
- equal borders on both sides

which are not necessarily true. thus further work will be needed... :+1: 
in case code repeats for x and y: consider to define a fuction and call it twice :smile: